src/Types.jl

```diff
@@ -997,8 +1008,7 @@ function clone_or_cp_registries(ctx::Context, regs::Vector{RegistrySpec}, depot:
         # clone to tmpdir first
         mktempdir() do tmp
             url, registry_urls = pkg_server_registry_url(reg.uuid, registry_urls)
-            # on Windows we prefer git cloning because untarring is so slow
-            if !Sys.iswindows() && url !== nothing
+            if registry_try_git(url)
```

Comment:
I believe the CI errors you saw where from the logic here
```suggestion
            if registry_try_git(url) && url !== nothing
```